tests/templates/test_subroutines/test_approx_time_evolution.py

```diff
@@ -29,25 +29,6 @@ def test_standard_validity():
     qml.ops.functions.assert_valid(op)
 
 
-# pylint: disable=protected-access
-def test_flatten_unflatten():
```

Comment:
Might be worth keeping this one around since it also tests the actual contents of of the outputs, and not just "can we unpack and repack".

Reply:
Right. Although the assert_valid check makes sure that an equivalent op is created from unflattening, right? So one could argue that this test failing while `assert_valid` passing "does not matter too much" ? :thinking:

Reply:
No strong preference here.